app/src/main/java/com/example/tour_a_versity_v3/MapsActivity.kt

```diff
@@ -127,36 +123,24 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback, GoogleMap.OnInfoWi
         )
 
     }
+
+    fun createCechMarker() {
```

Comment:
```suggestion
    private fun createCechMarker() {
```
I forgot to put `private` here.